src/help/guides/getting-started.md

```diff
@@ -63,42 +64,42 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
    <el-tag type="warning">Not the Windows Store app!</el-tag>
 1. Download and install [iCloud for Windows 8](https://support.apple.com/en-us/HT204283).
    <el-tag type="warning">Not the Windows Store app!</el-tag>
-   > Can't find the link to download **iCloud** for Windows 8? [Image](https://imgur.com/a/P1ef4Wd)
-1. Restart your PC after installing both **iTunes** and **iCloud**.
+  > Can't find the link to download **iCloud** for Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
+1. Restart your PC after you've installed both **iTunes** and **iCloud**.
    <el-tag type="warning">You need to restart your computer!</el-tag>
 1. Extract the downloaded **`AltInstaller.zip`** file. Run **`setup.exe`** and follow the instructions.
-1. Launch **AltServer** from your *Notification Area*/*System Tray* or *Hidden Icons*.
+1. Launch **AltServer** from your *Notification Area*.
 	> Can't find **AltServer**? [Image](https://imgur.com/a/rSagfh2)
```

Comment:
I would suggest to use the same word for all the links: `Image` or `GIF`. Or is there an intended difference between Image and GIF?

Reply:
The GIF's are the GIF's, you know, the video's.
Image is just for screenshots.

From UX design perspective, the user will know in this case if they get to see a still image or a video.

Reply:
Make sense! It seems judicious.